src/api/app/controllers/webui/webui_controller.rb

```diff
@@ -323,6 +324,15 @@ def set_pending_announcement
     @pending_announcement = Announcement.last
   end
 
+  def set_influxdb_tags
+    anonymous = User.current.present? && !User.current.is_nobody?
```

Comment:
[nitpick] repeated code :see_no_evil:

Reply:
I don't think it makes sense to abstract it now!